subprojects/packagefiles/cjson/meson.build

```diff
@@ -6,12 +6,17 @@ project('cjson', 'c',
 dllimport_args = []
 cc = meson.get_compiler('c')
 if cc.get_argument_syntax() == 'msvc'
-  add_project_arguments('-D_CRT_SECURE_NO_WARNINGS', cc.get_supported_arguments('-Wno-deprecated-declarations'), language: 'c')
```

Comment:
Why get rid of -Wno-deprecated-declarations?

Reply:
This flag a gcc/clang flag and not being recognized by MSVC:

```
cl : Command line error D8021 : invalid numeric argument '/Wdeprecated-declarations'
```

Reply:
hmm? cc.get_supported_arguments should fail for MSVC and succeed for clang-cl.

Reply:
My bad. I have reverted this line.